ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-master.yaml

```diff
@@ -1,40 +1,40 @@
 base_images:
-  ocp_builder_golang-1.13:
-    name: builder
+  base:
+    name: "4.11"
     namespace: ocp
-    tag: golang-1.13
-  ocp_builder_rhel-8-golang-1.17-openshift-4.10:
+    tag: base
+  ocp_builder_rhel-8-golang-1.17-openshift-4.11:
     name: builder
     namespace: ocp
-    tag: rhel-8-golang-1.17-openshift-4.10
+    tag: rhel-8-golang-1.17-openshift-4.11
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: rhel-8-release-golang-1.17-openshift-4.11
 images:
-- from: ocp_builder_golang-1.13
+- from: base
   inputs:
-    ocp_builder_rhel-8-golang-1.17-openshift-4.10:
+    ocp_builder_rhel-8-golang-1.17-openshift-4.11:
       as:
-      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.11
   to: sandboxed-containers-operator
 operator:
   bundles:
   - as: sandboxed-containers-operator-bundle
     dockerfile_path: bundle-custom.Dockerfile
   substitutions:
-  - pullspec: controller:latest
+  - pullspec: proxy.engineering.redhat.com/rh-osbs/openshift-sandboxed-containers-operator
```

Comment:
should that be registry-proxy.engineering.redhat.com instead?

Reply:
yes